sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLCommandTestUtils.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.execution.command
+
+import org.scalactic.source.Position
+import org.scalatest.Tag
+
+import org.apache.spark.sql.Row
+import org.apache.spark.sql.execution.datasources.PartitioningUtils
+import org.apache.spark.sql.test.SQLTestUtils
+
+trait DDLCommandTestUtils extends SQLTestUtils {
+  // The version of the catalog under testing such as "V1", "V2", "Hive V1".
+  protected def version: String
+  // Name of the command as SQL statement, for instance "SHOW PARTITIONS"
+  protected def command: String
+  protected def catalog: String
+  protected def defaultUsing: String
+
+  override def test(testName: String, testTags: Tag*)(testFun: => Any)
+    (implicit pos: Position): Unit = {
+    super.test(s"$command $version: " + testName, testTags: _*)(testFun)
+  }
+
+  protected def withNamespaceAndTable(ns: String, tableName: String, cat: String = catalog)
```

Comment:
very nit: the parameter names for namespace and catalog are both very short(`ns` and `cat`), we should probably use `t` to be consistent.

Reply:
If the build fails, I will change this to re-trigger GA or Jenkins.